assets/sass/_modal.sass

```diff
@@ -0,0 +1,41 @@
+.contributots-list
```

Comment:
I don't like current name for this css code. It is specific code only for one modal